codestodians/komlie-seller
# [FEAT] Clean seller repo and fix login

#### What does this PR do?
This PR removes redux from the seller repo alongside incorporating the use of apollo cache

#### Description of Task to be completed?

- Initial remove of redux structure from files - actions, reducers, store and the provider
- Manage change of username on login with the apollo cache
- Remove unused components directory
- Remove unused code/ redundant code
- Update test files

#### How should this be manually tested?
- Login an existing user or signup a new user
- The username should appear on the navbar
- On logout, the login button should change to the logout button

#### Any background context you want to provide?
There may be traces of redux left in the application as it stands, leaving it for now because of my bias towards file changes. I hace hit 30 and do not want to exceed that.
 
#### What are the relevant pivotal tracker stories?
None

#### Screenshots (if appropriate)
<img width="181" alt="Screenshot 2019-06-17 at 10 28 39 AM" src="https://user-images.githubusercontent.com/20999363/59593933-bf6f4c00-90ea-11e9-8342-d88f23ed0489.png">
<img width="254" alt="Screenshot 2019-06-17 at 10 28 33 AM" src="https://user-images.githubusercontent.com/20999363/59593934-c007e280-90ea-11e9-93f1-e0b8f27fa67e.png">

#### Questions:
